### PR TITLE
feat: Add ability to use PUT method for HTTP Export in configurable pipeline

### PR DESCRIPTION
**Pull request by lenny-intel**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number:  #128

## What is the new behavior?

Http Export using PUT method now available via the HTTPPut pipeline function

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

## Are there any new imports or modules? If so, what are they used for and why?

no
## Other information

### CODE REVIEW

File: res/http-export/configuration.toml [hunk 1]
```diff
@@ -26,6 +26,12 @@
         persistOnError = "false"
         secretheadername = "" # This is the name used in the HTTP header and also used as the secret key
         secretpath = ""
+    [Writable.Pipeline.Functions.HTTPPutJSON]
+      [Writable.Pipeline.Functions.HTTPPutJSON.Parameters]
+        url = "http://"
```
Comment (jim-wang-intel) [inline res/http-export/configuration.toml:31]: why this url only has protocol but no address?

Comment (lenny-intel) [reply res/http-export/configuration.toml:31]: @jim-wang-intel , It is just a place holder for the user to change to their actual URL.

Comment (jim-wang-intel) [reply res/http-export/configuration.toml:31]: got it, maybe add some comments to indicate that like " # Add your own url or modify this"

**Review by jim-wang-intel — APPROVED** [review]
LGTM

**Pull request merged**